lib/psi.rb

```diff
@@ -3125,6 +3125,22 @@ def processTBD(
     end
   end
 
+  # Ensure deratable surfaces have U-factors (even if NOT derated).
+  surfaces.each do |id, surface|
+    next unless surface.has_key?(:deratable)
+    next unless surface[:deratable]
+    next unless surface.has_key?(:construction)
+    next if surface.has_key?(:u)
```

Comment:
Most opaque envelope surfaces get their U-factors (or `:u`) set when _thermally derating_ their targeted insulation materials. Some surfaces (e.g. those without peripheral _edges-as-thermal-bridges_) don't get derated at all (which is fine in itself) and so didn't get their U-factors set. This generates errors in UA' summaries (i.e. has no bearing on energy simulation results). Simple fix.